tests/virt_autotest/guest_installation_run.pm

```diff
@@ -64,6 +65,13 @@ sub post_execute_script_assertion {
 sub run {
     my $self = shift;
 
+    # refer to poo#135431 for more details
+    my $proxyscc = get_var('SCC_URL');
```

Comment:
I happen to think of a problem that the versions of guests and hosts are often different. `SCC_URL` is set based on the version of the host. ie. `SCC_URL=http://all-20.1.proxy.scc.suse.de/` for sle15sp6 host, and `SCC_URL='' ` for sle12sp5/sle15sp5 hosts.

So we can't use SCC_URL directly for guests. UEFI tests have the same problem.

So the solution I think out is to change the guest autoyast profile dirrectly to add the proxy server for sle15sp6 guests(or even turn of registration). For UEFI tests, adding the proxy server(or turn off registration) in the guest paramter files. 

@alice-suse  @waynechen55 @guoxuguang

Reply:
Great point, confirm that 15SP6 guest migration test run have the same problem for now. 

So, sum up the following solution:
1. Disable scc registration for 15sp6 guest installation 
    Create a new [PR#803](https://github.com/SUSE/qa-automation/pull/803) - [qa_lib_virtauto-data] disable scc registration for 15sp6 guest installation  
 2. Continue to use proxyscc for 15sp6 guest installation
     Create a new [PR#802](https://github.com/SUSE/qa-automation/pull/802) -  [qa_lib_virtauto-data] use proxyscc for 15sp6 guest installation

Reply:
>So the solution I think out is to change the guest autoyast profile dirrectly to add the proxy server for sle15sp6 guests(or even turn off registration). For UEFI tests, adding the proxy server(or turn off registration) in the guest paramter files.

I am fine with whatever works and valid :)